src/main/scala/org/ergoplatform/api/routes/InfoRoute.scala

```diff
@@ -71,7 +72,7 @@ object InfoRoute {
       "stateRoot" -> stateRoot.asJson,
       "difficulty" -> bestFullBlock.map(_.header.requiredDifficulty).getOrElse(BigInt(0)).toString(10).asJson,
       "unconfirmedCount" -> unconfirmedCount.asJson,
-      "stateType" -> stateType.asJson,
+      "stateType" -> stateType.toString.asJson,
```

Comment:
You have an `Encoder` for `StateType` so `stateType.asJson` should work?

Reply:
yes, this would work, you're really sharp. I left `toString` here just for the better safety but `circe` does this job good enough.